### ISSUE TITLE
Fixes needed for builds on WCOSS Dell

### ISSUE THREAD

**Issue by BenjaminBlake-NOAA** [NOAA-EMC/regional_workflow#285]
1. The settings for the ufs-weather-model build on the Dell need to be updated
2. Add ability to see make output from ufs-weather-model to see what compiler flags are used
3. Build of chgres fails in UFS_UTILS_chgres_grib2/sorc/chgres_cube.fd
4. Build of orog fails in build_orog.sh
5. Build of regional_grid fails

Note that #3-5 may be fixed when the workflow is updated to use the latest UFS_UTILS

**Comment by jwolff-ncar** [comment]
@BenjaminBlake-NOAA Assuming the UFS SRW App v1 release will depend on this being completed - is that true?

**Comment by BenjaminBlake-NOAA** [comment]
@jwolff-ncar  Yes that's correct, we will want it to compile/build on WCOSS.  I'm hoping the fixes won't be too difficult since I ran end-to-end tests prior to the big merge.  And as I noted above, #3-5 may be fixed when the UFS_UTILS commit goes in.

**Comment by BenjaminBlake-NOAA** [comment]
@JeffBeck-NOAA @gsketefian   I was able to compile the ufs-weather-model code on the Dell, but I am wondering if #1 is related to pointing to the NCAR fork of ufs-weather-model.  Jim Abeles is the one who pointed out #1 and #2.  On WCOSS we usually point to the latest develop code in ufs-community/ufs-weather-model, so the WCOSS build settings have likely been updated there

**Comment by BenjaminBlake-NOAA** [comment]
PR #274 for regional workflow and PR #17 for ufs-srweather-app fixes issues #3-5.  I can confirm that the executables build successfully on the WCOSS Cray and Dell!

**Comment by JeffBeck-NOAA** [comment]
@BenjaminBlake-NOAA Thanks for checking on this!  I plan to issue a PR to move the Externals.cfg file in the develop branch of ufs-srweather-app to the HEAD of all repositories.  We will be creating a "stable" branch that will point to specific hashes in the future for a known working version.

**Comment by JeffBeck-NOAA** [comment]
@BenjaminBlake-NOAA Could you, @RatkoVasic-NOAA, or @JamesAbeles-NOAA, change the Externals.cfg in ufs-srweather-app to the following, run manage_externals, and try building again on WCOSS:

repo_url = https://github.com/ufs-community/ufs-weather-model
branch = release/public-v2

**Comment by BenjaminBlake-NOAA** [comment]
@JeffBeck-NOAA  Yes I can try that this morning

**Comment by BenjaminBlake-NOAA** [comment]
@JeffBeck-NOAA  The ufs-weather-model code from the release/public-v2 branch builds successfully on the WCOSS Cray and Dell

**Comment by JeffBeck-NOAA** [comment]
Thanks, @BenjaminBlake-NOAA!  I'll report this during our tag up meeting this morning.  Once the UFS_UTILS PRs for the regional_workflow and the umbrella repo are merged, we can then test that build on WCOSS and hopefully be able to close this issue.

**Comment by BenjaminBlake-NOAA** [comment]
No problem!  Sound great, and yeah we should be able to.

**Comment by BenjaminBlake-NOAA** [comment]
This issue can be closed.
